pkg/qliksense/schema/qliksense.json

```diff
@@ -24,21 +24,7 @@
     "invokeStep": {
       "type": "object",
       "properties": {
-        "qliksense": {
-          "type": "object",
-          "properties": {
-            "version": {
-              "type": "string"
-            },
-            "description": {
-              "$ref": "#/definitions/stepDescription"
-            }
-          },
-          "additionalProperties": false,
-          "required": [
-            "version"
-          ]
-        }
+        "qliksense": {"$ref": "#/definitions/qliksense"}
```

Comment:
Rather than maintaining a global schema for customSteps, we should just use a specific one. I merged some schema changes you may want to pick up that gets rid of invokeStep.

Reply:
Rolling back changes to the schema.